default.nix

```diff
@@ -1,6 +1,8 @@
 { nixpkgsFunc ? import ./nixpkgs
 , system ? builtins.currentSystem
-, config ? {}
+, config ? {
+    android_sdk.accept_license = true;
+  }
```

Comment:
We only do this in `release.nix` on purpose, as we cannot opt into the license on behalf of our users.

Reply:
Feel free to add some sort of comment so this is clear to the next person to edit this file.

Reply:
Sorry, leftover from local testing.